platform/linux-generic/m4/odp_scheduler.m4

```diff
@@ -4,8 +4,8 @@
 AC_DEFUN([ODP_SCHEDULER], [dnl
 AC_ARG_ENABLE([scheduler-default],
 	      [AS_HELP_STRING([enable-scheduler-default],
-			      [Choose default scheduler (default is basic)])],
+			      [choose default scheduler [default=basic] (linux-generic)])],
 	      [], [enable_scheduler_default=basic])
 AC_DEFINE_UNQUOTED([_ODP_SCHEDULE_DEFAULT], ["$enable_scheduler_default"],
-		   [Define to name default scheduler])
+		   [define to name default scheduler])
```

Comment:
No need to change this. Used for config.h (autoheader_build.h) generation, where descriptions start with capital letters.